addons/misc/CfgWeapons.hpp

```diff
@@ -55,4 +55,17 @@ class CfgWeapons
         editorPreview = QPATHTOF(ui\ArmbandKat.paa);
         picture = QPATHTOF(ui\ArmbandKat.paa);
     };
+
+    class kat_Bubble_Wrap_Item: kat_Armband_Red_Cross_Item {
```

Comment:
This should not inherit from the kat_Armband_Red_Cross_Item, but rather ItemCore